src/api/get-browser-info.js

```diff
@@ -0,0 +1,42 @@
+import getInstallations from './get-installations';
+import exists from '../utils/fs-exists-promised';
+import { MESSAGES, getText } from '../messages';
+import ALIASES from '../aliases';
+
+
+//Const
+const DEFAULT_ALIAS = {
+    cmd:                '',
+    macOpenCmdTemplate: 'open -n -a "{{{path}}}" --args {{{pageUrl}}}'
+};
+
+
+//API
+/**
+ * Returns information about the browser specified by browser alias ('chrome', 'ff')
+ * or a path to browser executable file.
+ * @function
+ * @async
+ * @name getBrowserInfo
+ * @param {string} browser - Browser alias or a path to browser executable file.
+ * @returns {BrowserInfo} Object containing information about the browsers installed on the machine.
+ */
+export default async function (browser) {
+    var installations = await getInstallations();
+
+    var browserAsAlias = browser.trim().toLowerCase();
+
+    if (installations[browserAsAlias])
+        return installations[browserAsAlias];
+
+    var fileExists = await exists(browser);
+
+    if (!fileExists)
+        throw new Error(getText(MESSAGES.unableToFindBrowser, browser));
```

Comment:
It more like `MESSAGES.browserPathDoesntExist` or something like that. \cc @VasilyStrelyaev